content/authors/elaineliwang/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+# Display name
+title: Li Wang
+
+# Username (this should match your github username)
+authors:
+- elaineliwang
+
+superuser: false
+
+# Role/position e.g. Graduate student
+role:  GRADUATE STUDENT
+
+# Organization membership
+Organizations/Affiliations:
+organizations:
+- name:  University of British Columbia
+  url:  http://www.math.ubc.ca/~lwang/
+
+# A list of your interests, add as many as you want
+interests:
+-  Partial Differential Equations
+-  
+
+# Your educational history, add as many as you want
+education:
+  courses:
+  - course: B.Sc in Mathematics
+    institution: University of British Columbia
+
+# Social/Academic Networking, delete any items you don't want to use
+social:
+- icon: twitter
+  icon_pack: fab
+  link: 
+- icon: github
+  icon_pack: fab
+  link: 
+user_groups:
+- committee: members
+  role: Project Member
```

Comment:
Can you change "Project Member" to say what project you are in, e.g. change it to "ATCO Project Member" if you are in the ATCO project.

Reply:
Thanks! @ianabc it is fixed!